frontend/public/components/utils/dropdown.jsx

```diff
@@ -445,7 +447,71 @@ class Dropdown_ extends DropdownMixin {
 
     _.each(items, (v, k) => addItem(k, v));
     //render PF4 dropdown markup if this is not the autocomplete filter
```

Comment:
Does this comment need to be updated?

Reply:
I'm not sure myself, if this was PF4 specific the block may not be needed?